node/src/command.rs

```diff
@@ -160,48 +169,39 @@ pub fn run() -> Result<()> {
         }
         Some(Subcommand::CheckBlock(cmd)) => {
             construct_async_run!(|components, cli, cmd, config| {
-				Ok(cmd.run(components.client, components.import_queue))
-			})
+                Ok(cmd.run(components.client, components.import_queue))
+            })
         }
         Some(Subcommand::ExportBlocks(cmd)) => {
-            construct_async_run!(|components, cli, cmd, config| {
-				Ok(cmd.run(components.client, config.database))
-			})
+            construct_async_run!(|components, cli, cmd, config| { Ok(cmd.run(components.client, config.database)) })
         }
         Some(Subcommand::ExportState(cmd)) => {
-            construct_async_run!(|components, cli, cmd, config| {
-				Ok(cmd.run(components.client, config.chain_spec))
-			})
+            construct_async_run!(|components, cli, cmd, config| { Ok(cmd.run(components.client, config.chain_spec)) })
         }
         Some(Subcommand::ImportBlocks(cmd)) => {
             construct_async_run!(|components, cli, cmd, config| {
-				Ok(cmd.run(components.client, components.import_queue))
-			})
+                Ok(cmd.run(components.client, components.import_queue))
+            })
         }
         Some(Subcommand::PurgeChain(cmd)) => {
             let runner = cli.create_runner(cmd)?;
 
             runner.sync_run(|config| {
                 let polkadot_cli = RelayChainCli::new(
                     &config,
-                    [RelayChainCli::executable_name().to_string()]
```

Comment:
many of these rustfmt changes are only going to make it more time consuming for us to maintain our own codebase, as we'll be having to pattern match changes that are made to the substrate-node-template, and the more we deviate from it the harder our job becomes.

i'd suggest all rustfmt-specific changes go in a separate PR to https://github.com/substrate-developer-hub/substrate-parachain-template, and if only apply the rustfmt changes that they made to our codebase.